docs/supported-version.md

```diff
@@ -6,9 +6,10 @@ Xen Orchestra is designed to work exclusively on [XCP-ng](https://xcp-ng.org/) a
 
 Backup restore for large VM disks (>1TiB usage) is [broken on all XenServer versions](https://bugs.xenserver.org/browse/XSO-868) until Citrix release a fix.
 
+- Citrix Hypervisor 8.1
+- Citrix Hypervisor 8.0
 - XenServer 7.6
 - XenServer 7.5
-  - [VDI I/O error](https://bugs.xenserver.org/browse/XSO-873), waiting for Citrix to release our fix
```

Comment:
Why?

Reply:
Well there is a message from the 13-08-2018 saying:
> Fixed; will be available in the upcoming release.

I assumed it was fixed.

Reply:
7.5.1 or 7.6?

Unless absolutely sure it's fixed I think it's better to leave this warning.

Reply:
You are right - 
```
Fix Version/s:
7.6
```

Reply:
@julien-f ?

Reply:
Please revert your change.

Reply:
done